### PR TITLE
Changes in VAE training and MorphoMNIST

### PR DESCRIPTION
**Pull request by AnejSvete**
This PR contains a few modifications to the existing code to produce/train VAEs for the MIST publication.

### CODE REVIEW

File: latte/models/benchmarks/conv_models.py [hunk 1]
```diff
@@ -17,13 +18,13 @@
...
 
-    def __init__(self, model_config: BaseTrainerConfig) -> None:
+    def __init__(self, model_config: BaseTrainerConfig, n_channels: int = 3) -> None:
         BaseEncoder.__init__(self)
 
         self.model_config = model_config
 
         self.encoder = nn.Sequential(
-            nn.Conv2d(3, 32, 4, 2, 1),  # B, 32, 32, 32
+            nn.Conv2d(n_channels, 32, 4, 2, 1),  # B, 32, 32, 32
```
Comment (pawel-czyz) [inline latte/models/benchmarks/conv_models.py:27]: Good change!

File: scripts/train_pythae_vae.py [hunk 2]
```diff
@@ -136,7 +141,7 @@ def _get_model_classes(dataset: str, network_type: Optional[str] = None) -> Tupl
         The encoder and decoder class.
     """
     if dataset == "dsprites":
-        return None, None  # Not implemented yet
+        return lambda c: conv_models.ConvEncoder(c, n_channels=1), lambda c: conv_models.ConvDecoder(c, n_channels=1)
```
Comment (pawel-czyz) [inline scripts/train_pythae_vae.py:144]: Somewhat hacky, but I see the reason :slightly_smiling_face:

**Review by pawel-czyz — APPROVED** [review]
Nice! Please, merge :slightly_smiling_face:

File: scripts/train_pythae_vae.py (hunk 2 above)
Comment (AnejSvete) [reply scripts/train_pythae_vae.py:144]: Yes :D

**Pull request merged**